.github/workflows/test.yml

```diff
@@ -1,13 +1,13 @@
 name: build
-on: [push, pull_request]
+on: [push, pull_request_target]
```

Comment:
Why was this changed? It makes running tests in pull requests execute against the wrong branch(base branch context instead of pull request branch context).